tools/python_api/src_cpp/pyarrow/pyarrow_scan.cpp

```diff
@@ -12,9 +13,28 @@ using namespace kuzu::catalog;
 
 namespace kuzu {
 
-static std::unique_ptr<function::TableFuncBindData> bindFunc(main::ClientContext* /*context*/,
-    TableFuncBindInput* input) {
+PyArrowScanConfig::PyArrowScanConfig(const std::unordered_map<std::string, Value>& options) {
+    skipNum = 0;
+    limitNum = NumericLimits<uint64_t>::maximum();
+    for (const auto& i : options) {
+        if (i.first == "SKIP") {
+            if (i.second.getDataType().getLogicalTypeID() != LogicalTypeID::INT64) {
```

Comment:
I think the skip datatype should always be a postive number. What if the user gives a negative number? Do we error?

Reply:
Yeah I'll Implement this for both skip and limit. I'm not sure if an issue is open on it currently, but our parser can't handle negative parameters at the moment.